Tools/scripts/summarize_stats.py

```diff
@@ -442,7 +443,7 @@ def get_gc_stats(self) -> list[dict[str, int]]:
             gc_stats[gen_n][name] = value
         return gc_stats
 
-    def get_optimization_stats(self) -> dict[str, tuple[int, int | None]]:
+    def get_optimization_stats(self) -> dict[Doc, tuple[int, int | None]]:
```

Comment:
This is also unrelated, right?

Reply:
I believe it is related, as the fix for the issue is to treat the value returned from this function as a doc and not a str, so it makes the patch clearer